renovate.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "enabledManagers": ["github-actions", "jsonata", "npm", "nvm"],
```

Comment:
Do we need `jsonata`?

Reply:
Don't know, that was already in there. The line is shown as a diff, since I moved it up -- I like alphabetical ordering if at all possible 😁
@setchy Is there a reason you put it in there?

Reply:
@prabhu I just noticed that @setchy added in #1895.